02-atmos/components/terraform/output-results/main.tf

```diff
@@ -0,0 +1,63 @@
+variable "print_users_weather_enabled" {
+  default     = false
+  type        = bool
+  description = "Whether or not to pretty print the weather results"
+}
+
+data "terraform_remote_state" "weather" {
+  backend = "local"
+
+  config = {
+    path = "${path.root}/../fetch-weather/terraform.tfstate.d/example/terraform.tfstate"
+  }
+}
+
+data "terraform_remote_state" "location" {
+  backend = "local"
+
+  config = {
+    path = "${path.root}/../fetch-location/terraform.tfstate.d/example/terraform.tfstate"
+  }
+}
+
+locals {
+  users_location = data.terraform_remote_state.location.outputs.users_location
+
+  weather_data        = data.terraform_remote_state.weather.outputs.weather_data
+  weather_temp        = local.weather_data.main.temp
+  weather_description = local.weather_data.weather[0].description
+}
+
+resource "null_resource" "print" {
```

Comment:
template file seems more natural?

Reply:
@osterman I'm good with that, but how do we want to output that template file after generating it? A separate `null_resource` + `local-exec` to read it and `echo` out?

Reply:
It can just be an output too, no? In the end, do what looks better =)